web/app.js

```diff
@@ -1047,17 +1035,11 @@ let PDFViewerApplication = {
         return;
       }
       pdfDocument.getJavaScript().then((javaScript) => {
-        if (javaScript.length === 0) {
+        if (!javaScript) {
           return;
         }
-        javaScript.some((js) => {
-          if (!js) { // Don't warn/fallback for empty JavaScript actions.
-            return false;
-          }
-          console.warn('Warning: JavaScript is not supported');
-          this.fallback(UNSUPPORTED_FEATURES.javaScript);
-          return true;
-        });
```

Comment:
The `javaScript.some((js) => { ... });` changes shouldn't be reverted (note PR #9009).